src/components/layout/BreakingNews.jsx

```diff
@@ -19,6 +20,14 @@ const BreakingNews = ({ firstArticle: article }) => {
             {article.title}
           </h2>
           <h4 data-cy='breaking-teaser'>{article.teaser}</h4>
+          <Rating
+            data-cy='breaking-rating'
+            defaultRating={Math.floor(article.rating)}
```

Comment:
and here, maybe make a function for that?

Reply:
Hm ye, but wouldn't we just be repeat the function call instead? But true, the naming of a function would make it more clear as to what we're doing